lib/rubocop/cop/metrics/method_length.rb

```diff
@@ -12,18 +12,22 @@ class MethodLength < Cop
         LABEL = 'Method'.freeze
 
         def on_def(node)
+          return if excluded_methods.include?(String(node.method_name))
           check_code_length(node)
         end
         alias on_defs on_def
 
         def on_block(node)
           return unless node.send_node.method_name == :define_method
-
           check_code_length(node)
         end
 
         private
 
+        def excluded_methods
+          cop_config['ExcludedMethods'] || []
```

Comment:
I don't think you need `|| []`. We get an empty array from the default configuration if it hasn't been overridden.

Reply:
Removed.